chains/solana/contracts/programs/ccip-offramp/src/instructions/v1/execute.rs

```diff
@@ -522,6 +658,10 @@ pub fn validate_execution_report<'info>(
         commit_report.timestamp != 0,
         CcipOfframpError::RootNotCommitted
     );
+    require!(
+        commit_report.chain_selector == execution_report.source_chain_selector,
```

Comment:
Please add a comment clarifying that this can't easily be done in the context, as the report may be buffered.
Also, please use `require_eq!` when possible instead (and I see a few other checks in this function that could also use that other macro, heh), as that gives more info if the error ever arises, making the troubleshooting a bit easier